cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -209,7 +224,15 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate rules
             }
         }
 
-        val evaluator = Evaluator(ortResultInput, packageConfigurationProvider, licenseConfiguration)
+        val copyrightGarbage = copyrightGarbageFile.takeIf { it.isFile }?.readValue<CopyrightGarbage>().orEmpty()
```

Comment:
Can `takeIf { it.isFile }` be omit due to `canBeFile = true, canBeDir = false` and `mustBeReadable = true`?